### PR TITLE
feat: add i18n metadata for palette items #492

### PR DESCRIPTION
**Pull request by vadimbasko**
affects: @amplicode/codegen

### CODE REVIEW

File: example-app/src/dev/rb-metadata.json [hunk 1]
```diff
@@ -146,6 +177,263 @@
...
+    "Rate": {
+      "messageProps": []
+    },
+    "Select": {
+      "messageProps": [
+        "placeholder"
+      ]
+    },
+    "Slider": {
+      "messageProps": []
```
Comment (web-devel) [inline example-app/src/dev/rb-metadata.json:231]: Why it's an empty array?

File: example-app/src/dev/rb-metadata.json [hunk 2]
```diff
@@ -146,6 +177,263 @@
...
+    },
+    "Switch": {
+      "messageProps": []
```
Comment (web-devel) [inline example-app/src/dev/rb-metadata.json:234]: Why it's an empty array?

**Review by web-devel — CHANGES_REQUESTED** [review]
I suggest to remove empty `messageProps`.
Also I thought there will be more localisable props (but it was just an assumption)

File: example-app/src/dev/rb-metadata.json (hunk 2 above)
Comment (vyacheslav-pushkin) [reply example-app/src/dev/rb-metadata.json]: Removed empty arrays. Checked several components in the docs - seems that message props are defined correctly.

**Review by web-devel — APPROVED** [review]

**Pull request merged**